src/utils/formatSelectOutput.ts

```diff
@@ -0,0 +1,82 @@
+import { ObjectFieldSchema, Schema } from '../types';
+
+function formatSelect(
+  value:
+    | undefined
+    | { value: string; label: string }
```

Comment:
Can we define a type for this, or maybe use an existing type from `react-select`?